app/main.py

```diff
@@ -2,0 +1,12 @@
+def calculate_team_total_rating(team: int) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: int) -> None:
```

Comment:
The parameter `elves` should be a list of `Elf` objects, not an integer. Update the type annotation to `List[Elf]`.